helm/hpcc/templates/_helpers.tpl

```diff
@@ -247,6 +247,10 @@ cost:
 logAccess:
 {{ toYaml .Values.global.logAccess | indent 2 }}
 {{- end }}
+{{- if .Values.global.misc }}
```

Comment:
misc doesn't seem the most descriptive name, and it is currently used for something unrelated.  I'm not sure that would be better.  "global.options", "global.config", "storage.options"?  The latter may cause problems for bare-metal.
Not sure I have a strong preference, but worth thinking a bit more about - especially if more options might be added later.